schema/soo/ofc/custom_properties/CT_Property.go

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 FoxyUtils ehf. All rights reserved.
+// Copyright 2017 FoxyUtils ehf. All rights reserved.
```

Comment:
What triggered going back in time?  Is that normal?

Reply:
Looks like we made these 2019 changes on temporary unidoc-schema. They will be anyway changed when we change it to 2020 with a separate PR which you had told about